mesonbuild/msetup.py

```diff
@@ -52,10 +52,6 @@ def add_arguments(parser: argparse.ArgumentParser) -> None:
                         default=[],
                         action='append',
                         help='File describing cross compilation environment.')
-    parser.add_argument('--vsenv', action='store_true',
```

Comment:
Should it be deprecated before removal? Should it be documented that this options is removed?

Reply:
`--foo` is automatically added for all builtin options alongside `-Dfoo`, so that's backward compatible.